### DISCUSSION TITLE
How to enable clipboard inside the sandbox

### DISCUSSION THREAD

**Discussion by clippy-sbie** [sandboxie-plus/Sandboxie#1724] [Q&A Feedback]
Pardon me if this info can be found somewhere, I did not find it.

I use Sandboxie Plus 1.0.14 on Win10x64 to run Mozilla Firefox, and I am able to copy non-sandboxed clipboard content into the sandboxed browser. However, I am not able to copy inside sandboxed browser windows and paste into either other sandboxed browser tabs or any non-sandboxed windows. 

I often need to copy links or text from my sandboxed browser into non-sandboxed windows. This was always possible in Sandboxie Classic.

How do I enable this? Thanks in advance!

Also, might this be related to moving tabs around inside Firefox by Drag & Drop? That is currently not working either.

**Comment by isaak654** [comment]
> I often need to copy links or text from my sandboxed browser into non-sandboxed windows.
> Also, might this be related to moving tabs around inside Firefox by Drag & Drop? That is currently not working either.

I can't reproduce them with Plus 1.0.15 x64 and Firefox 98.0.2 x64 (one sandboxed and one non-sandboxed simultaneously) on Win10 x64.

> This was always possible in Sandboxie Classic.

It would be useful to know your last working version by opening a new problem report [here](https://github.com/sandboxie-plus/Sandboxie/issues/new/choose).
